src/runner.ts

```diff
@@ -601,9 +601,9 @@ export class Runner {
             result.workspaceRepo.githubRepoRef)}`;
         if (this._color) {
           const colorFunction = {
-              'PASSED': chalk.green,
-              'SKIPPED': chalk.yellow,
-              'FAILED': chalk.inverse.red
+            'PASSED': chalk.green,
```

Comment:
It seems like somehow formatting rules changed or my computer started formatting properly.